### DISCUSSION TITLE
How to add Header to Out-HtmlView

### DISCUSSION THREAD

**Discussion by davesbrown** [EvotecIT/PSWriteHTML#295] [Q&A]
I see New-Html has param, how can I add a Header with Out-HtmlView?

Thanks

**Comment by PrzemyslawKlys** [comment]
Whatever is applicable to New-HTMLTable is applicable to Out-HtmlView.

You can see New-HTMLTable in action:
- https://evotec.xyz/all-your-html-tables-are-belong-to-us/
- https://evotec.xyz/advanced-html-reporting-using-powershell/

And you can use the options shown there in Out-HTMLView. If that's what you're asking?
